DataHub/UserInterface/02b_AttributeSets.md

```diff
@@ -0,0 +1,397 @@
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all attributes in the list are selected.
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)   
+  Click this button to edit the configuration of the selected attribute. This button is only displayed if a single checkbox in the list of attributes is selected and if the selected attribute has an editable configuration. The *Edit attribute for set "Attribute set name"* view is displayed. Alternatively, you can click directly a row in the list to edit an attribute configuration.
+
+[comment]: <> (Edit is not working -> Bugticket)
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)   
+  Click this button to remove the selected attribute from the attribute set. This button is only displayed if at least one checkbox of an attribute is selected.       
+
+  > [Info] If you remove an attribute from the attribute set, the attribute itself is not deleted, but it is unassigned to the selected attribute set.   
+
+- [ADD MAPPING]   
+  Click this button to add a mapping to the selected attribute. The *Add mapping* view is displayed. You can only add a mapping for attribute sets that are not inherited from another attribute set. This button is only displayed if a single checkbox of an attribute is selected.
+
+
+The list displays all attributes assigned to the selected attribute set. Depending on the settings, the displayed columns may vary.   
+If the ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings) button is gray, all fields are read-only. If the ![Settings](../../Assets/Icons/Settings02.png "[Settings]") (Settings) button is blue, the field *Required* is editable.
+
+- *Name*   
+  Attribute name.
+
+
+- *Calculated field*   
+  Indication whether the attribute is calculated by an ETL mapping from other fields to itself or not:
+  - ![Check](../../Assets/Icons/Check.png "[Check]") (Checkmark): The attribute is a calculated field.   
+  - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross): The attribute is no calculated field.
+
+
+- *Key*   
+  Attribute key.
+
+- *Data type*   
+  Attribute data type.  
+  For detailed information about all data types, see [Data type list](./04_DataTypeList.md).
+
+- *Inherited from*   
+  Inheritance attribute set. The name of the attribute set from which the attribute value is inherited is displayed. If the attribute value is not inherited, the field is empty.
+
+- *Plugin*   
+  Module or plugin that owns the attribute set.
+
+
+- *Required*  
+  Indication whether the attribute value is required or not. The following options are only displayed if the ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings) button is inactive:
```

Comment:
```suggestion
  Indication whether the attribute value is required or not. The following options are available/possible:
```
he following options are only displayed if the ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings) button is inactive: -> Das ist schon in der zweiten Option aufgeführt, oder?